docs/native-modules-android.md

```diff
@@ -324,10 +447,28 @@ For argument types not listed above, you will need to handle the conversion your
 
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+    val dateFormat = "yyyy-MM-dd"
+    val sdf = SimpleDateFormat(dateFormat, Locale.US)
+    val eStartDate = Calendar.getInstance()
+    sdf.parse(startDate)?.let {
```

Comment:
Previously there was a `try` there that you removed. I'm assuming it was needed to catch parsing exceptions therefore should be kept here as well (maybe with a `runCatching` or something more idiomatic).